code/app/src/client/temp/TypographicScale.jsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import Component from 'react-pure-render/component';
+import './TypographicScale.scss';
+import Col from '../atoms/Col.jsx';
+import Row from '../atoms/Row.jsx';
+
+//generates type sizes array using base sie, scale and amount of sizes below and above scale
+function generateTypeSizesRem({scale = 1, below = 0, above = 0}) {
+  const base = 1; //all size relative to 1rem
+  let sizes = [];
+  let currentScale = base;
+
+  for (let i = 0; i < below; i++) {
+    currentScale /= scale;
+    sizes.unshift(currentScale);
+  }
+
+  sizes.push(base);
+
+  currentScale = base;
+  for (let i = 0; i < above; i++) {
+    currentScale *= scale;
+    sizes.push(currentScale);
+  }
+
+  return sizes.map(x => x.toFixed(3));
+}
+
+function remsToPx({sizeInRem, baseSizeInPx}) {
+  return Math.round((sizeInRem * baseSizeInPx));
+}
+
+class FontExample extends Component {
+  render() {
+    const {sizeInRem, baseSizeInPx, scale} = {...this.props};
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Expected an assignment or function call and instead saw an expression.](https://www.codacy.com/app/restuta8/rcn-io/file/1345712168/issues/source?bid=2967278&fileBranchId=2967430#l35)